srcpkgs/gnupg/patches/0001-use-versioned-pcsclite-library-on-musl-libc-as-well.patch

```diff
@@ -1,8 +1,8 @@
-From 5242588814474653bb0de15ebda3b0c6bb539ff6 Mon Sep 17 00:00:00 2001
+From 7f672f6281b0b27be8279cc5e422aeb55ffcf0fb Mon Sep 17 00:00:00 2001
 From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
  <congdanhqx@gmail.com>
 Date: Sun, 29 Sep 2019 12:32:46 +0700
-Subject: [PATCH] use versioned pcsclite library on musl-libc as well
+Subject: [PATCH 1/2] use versioned pcsclite library on musl-libc as well
```

Comment:
this patch rebase doesn't seem necessary, not even line numbers changed

Reply:
Well, I have a checkout of the `gnupg` source locally, and I generate the patchset with `git format-patch v${version}` after doing `git rebase v${version}`, so all of those are rebased.  Manually checking in which of those the context lines might have changed is additional churn that doesn't really add any value IMO, because having those updated patches in the repo doesn't really hurt anyway.. Is there a good reason to against this?

Reply:
I don't see any reason to rebase the patches even if the line numbers changed if they all still apply, it only makes the commit diffs bigger.

Reply:
I added a new patch, that's why I rebased the patchset. If there is no changes to the patchset at all, sure, I wouldn't rebase it.

Reply:
Please use `quilt` instead. Changing patches like this won't add anything but more objects into git storage.

Reply:
or you can check if nothing was changed in patches, please remove from changeset.

Reply:
I've removed the rebase of that patch from the changeset.